mlflow/store/file_store.py

```diff
@@ -87,12 +87,12 @@ def _get_experiment_path(self, experiment_id, view_type=ViewType.ALL):
         for parent in parents:
             exp_list = find(parent, str(experiment_id), full_path=True)
             if len(exp_list) > 0:
-                return exp_list
-        return []
+                return exp_list[0]
+        return None
```

Comment:
Should we raise an exception in this case actually? Otherwise we might try to use `None` as a path segment down the line in `build_path`, which will result in a more obscure `AttributeError: 'NoneType' object has no attribute 'endswith'`. The case where we use `None` as a path segment is admittedly rare right now (I think it can only happen if you call `log_xxxx` on an experiment that was manually deleted from the filesystem), so open to leaving as is, but seems safest to raise the exception here